lib/app/features/chat/messages/views/components/message_types/text_message/text_message.dart

```diff
@@ -33,15 +42,17 @@ class TextMessage extends StatelessWidget {
         child: Column(
           crossAxisAlignment: CrossAxisAlignment.start,
           children: [
-            RepliedMessageInfo(
-              isMe: isMe,
-              sender: ChatSender(
-                'Boris Johnson',
-                'https://picsum.photos/200/300',
-                isApproved: true,
+            MessageAuthorNameWidget(author: author),
+            if (repliedMessage != null)
+              RepliedMessageInfo(
+                isMe: isMe,
+                sender: const MessageAuthor(
+                  name: 'Boris Johnson',
+                  imageUrl: 'https://picsum.photos/200/300',
```

Comment:
Can we use real entity here, and mock name and url upper?

Reply:
You can create func in fake_posts_generator.dart

Reply:
Removed this mock and made provider with mocked data, so it placed in one place.